doc/ref/gappkg.xml

```diff
@@ -4,10 +4,10 @@
 <Index>package</Index>
 The  functionality  of  &GAP;  can  be  extended  by  loading  &GAP;
 packages. 
-&GAP; distribution already contains all currently redistributed with 
-&GAP; packages in the <F>&GAPDIRNAME;/pkg</F> directory.
+The &GAP; distribution contains, in the <F>&GAPDIRNAME;/pkg</F> 
+directory, all currently available &GAP; packages.
```

Comment:
available -> redistributed I think (there are packages not distributed with GAP)